# file: C:\Users\micha\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\coverage\data.py
# hypothesis_version: 6.138.15

['-journal', '.*', 'No data to combine', 'No usable data files', 'dataio', 'rb', 'sha3_256']